### DISCUSSION TITLE
Can rumble feature be enabled on GameCube?

### DISCUSSION THREAD

**Discussion by Detovar27** [darthcloud/BlueRetro#896] [Q&A]
I mean, I have my Ps5 controllers and I want to buy wired Gamcube controllers as well so I do them the wireless mod's. I'd like to know if there's an update that allows me to enable and/or activate the rumble feature on any of those. Thanks.

**Comment by darthcloud** [comment]
Go in the config to enable it:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#26---example-enable-rumble
